block/manager.go

```diff
@@ -114,6 +114,7 @@ func NewManager(
 	eventBus *cmtypes.EventBus,
 	logger log.Logger,
 	doneBuildingCh chan struct{},
+	blockStore *goheaderstore.Store[*types.Block],
```

Comment:
You are passing it in but not assigning it.

Reply:
Great catch, thanks!